### PR TITLE
zh-CN: update Glossary/Baseline/Compatibility

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/glossary/baseline/compatibility/index.md [hunk 1]
```diff
@@ -2,16 +2,28 @@
...
+  sourceCommit: 70aa041ebd7e31df76818bc8459f60659fe94c0c
 ---
 
 {{GlossarySidebar}}
 
-**基线**（Baseline）标识可跨浏览器使用的 Web 平台特性。基线可帮助你决定何时使用某项特性，告诉你该特性何时不太可能给网站访客带来兼容性问题。
+**基线**（Baseline）标识可跨浏览器使用的 Web 平台特性，包括 API、CSS 属性、JavaScript 语法。基线特性被描述为当前稳定浏览器中新提供的，或随着时间的推移可持续支持且广泛可用的特性。不符合基线标准的特性被描述为具有有限可用性。
 
-基线特性（如 API、CSS 属性集或 JavaScript 语法）可在 Chrome、Edge、Firefox 和 Safari 等多款知名且广泛使用的桌面和移动浏览器上一致运行。
+极限考虑以下浏览器的支持情况：
```
Comment (yin1999) [inline files/zh-cn/glossary/baseline/compatibility/index.md:12]:
```suggestion
基线考虑以下浏览器的支持情况：
```

File: files/zh-cn/glossary/baseline/compatibility/index.md [hunk 2]
```diff
@@ -2,16 +2,28 @@
...
+- Mozilla Firefox（Android）
+- Mozilla Firefox（桌面版）
+
+基线是对浏览器支持的概要。它并不是无障碍、可用性、性能、安全性或其他测试的替代品。基线可能无法告诉你特性是否能在以下情况下工作：
+
+- 较旧的设备和浏览器版本
+- 未包含在基线定义中的浏览器，例如操作系统的 WebView
+- 辅助技术，例如屏幕阅读器
 
 ## 标记
```
Comment (yin1999) [inline files/zh-cn/glossary/baseline/compatibility/index.md:28]: 下面这块的内容没更新

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/glossary/baseline/compatibility/index.md (hunk 1 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/baseline/compatibility/index.md:12]: Oops really sorry for that ridiculous typo

File: files/zh-cn/glossary/baseline/compatibility/index.md (hunk 2 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/baseline/compatibility/index.md:28]: 明天看看👌

**Review by yin1999 — APPROVED** [review]

**Pull request merged**